model/flow/constants.go

```diff
@@ -19,19 +19,28 @@ const DefaultTransactionExpiryBuffer = 30
 // DefaultMaxGasLimit is the default maximum value for the transaction gas limit.
 const DefaultMaxGasLimit = 9999
 
-// DefaultMaxTxSizeLimit is the default maximum transaction byte size. (1.5MB)
-const DefaultMaxTxSizeLimit = 1500000
+// DefaultMaxTxSizeLimit is the default maximum transaction byte size. (~1.5MB)
+const DefaultMaxTxSizeLimit = 1_500_000
+
+// DefaultMaxCollectionByteSize is the default maximum value for a collection byte size.
+const DefaultMaxCollectionByteSize = 3_000_000 // ~3MB. This is should always be higher than the limit on single tx size.
+
+// DefaultMaxCollectionTotalGas is the default maximum value for total gas allowed to be included in a collection.
+const DefaultMaxCollectionTotalGas = 10_000_000 // 10M
+
+// DefaultMaxCollectionSize is the default maximum number of transactions allowed inside a collection.
+const DefaultMaxCollectionSize = 100
 
 // DefaultAuctionWindow defines the length of the auction window at the beginning of
 // an epoch, during which nodes can bid for seats in the committee. Valid epoch events
 // such as setup and commit can only be submitted after this window has passed.
-const DefaultAuctionWindow = 50000
+const DefaultAuctionWindow = 50_000
 
 // DefaultGracePeriod defines the minimum number of views before the final view of
 // an epoch where we need to have an epoch setup and an epoch commit event. This is
 // in order to give all nodes the chance to have the information before entering
 // the next epoch.
-const DefaultGracePeriod = 25000
+const DefaultGracePeriod = 25_000
```

Comment:
These are unused I believe - we can just remove them

Reply:
are you sure we don't need them in the future?

Reply:
We won't need them.

Reply:
removed at 50368bb